src/App.js

```diff
@@ -2,14 +2,11 @@ import React from 'react';
 import InputBar from './components/InputBar';
```

Comment:
```
import React from 'react';

import Tabs from './components/Tabs';
import TodoBox from './components/TodoBox';
import InputBar from './components/InputBar';

import './assets/css/';
```

Separate third party package and our components. Also, using ladder formatting  in import.